### PR TITLE
Custom stack with ElasticSearch + custom file provisioning

### PR DESCRIPTION
**Pull request by keanolane**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,12 @@
-/vendor
+#
+## IDEs
+#
+.idea/
```
Comment (mikefrancis) [inline .gitignore]: I usually stick the `.idea` line in a global `.gitignore` as it's user specific, that way we don't have to add it to each repo.

File: resources/Homestead.yaml [hunk 2]
```diff
@@ -10,12 +10,32 @@ keys:
     - ~/.ssh/id_rsa
 
 folders:
-    - map: ~/Code
-      to: /home/vagrant/Code
+    - map: ~/Documents/Development/pod-point
```
Comment (mikefrancis) [inline resources/Homestead.yaml]: I think we should leave this folder as the default, not sure what everyone else uses as their root?

File: resources/after.sh [hunk 3]
```diff
@@ -3,3 +3,167 @@
...
+    echo " "
+    displayMessage "--------------------------------------"
+    echo "${WHITE_ON_GREEN}  ALL READY!  ${NC}"
+    echo " "
+    displayMessage "bower version: `bower -v`"
+    displayMessage "composer version: `composer -V | grep -Po '\d+(\.\d+)+'`"
+    displayMessage "elasticsearch version: `curl -s -XGET http://localhost:$ES_PORT | grep -E \"\\\"number\\\"\" | grep -Po '\d+(\.\d+)+'`"
+    displayMessage "php version: `php -v | grep -Po 'PHP \d+(\.\d+)+' | sed -e 's/PHP //'`"
+    displayMessage "mysql version: `mysql -Vmysql -V | grep -Po 'Distrib \d+(\.\d+)+' | sed -e 's/Distrib //g'`"
+    displayMessage "node version: `node -v`"
```
Comment (mikefrancis) [inline resources/after.sh:63]: I'm a bit wary of installing and using Node from homestead as there's other Node projects which don't depend on Homestead - such as the OpenChargeNative app etc. - think we should omit these bits from the set up.

File: resources/Homestead.yaml (hunk 2 above)
Comment (keanolane) [reply resources/Homestead.yaml]: We need to decide what the default will be for us then ;)

File: resources/after.sh (hunk 3 above)
Comment (keanolane) [reply resources/after.sh:63]: fair enough, can be removed

Comment (keanolane) [reply resources/after.sh:63]: but afair it's installed in homestead already

Comment (mikefrancis) [reply resources/after.sh:63]: Ah ok, ignore this comment then :)

**Pull request merged**